src/GraphBLAS-sharp.Backend/Matrix/Matrix.fs

```diff
@@ -0,0 +1,33 @@
+namespace GraphBLAS.FSharp.Backend
+
+open Brahma.FSharp.OpenCL
+open Microsoft.FSharp.Quotations
+
+module Matrix =
+    let toCSR (clContext: ClContext) workGroupSize processor matrix =
```

Comment:
Should these functions be wrapped to lambdas (like kernels) to prevent multiple compilations?